github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+Proposed labels by section on GitHub
+====================================
+>
+Assignees
+---------------------------------------------------------------
+
+> No labels for this section - just use the alias of the individuals owning the issue. MS alias showing is preferred.
```

Comment:
github aliases should and need to be used.  For PM purposes you can figure out behind the scenes the mapping